docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Button, Grid, Popup } from '@stardust-ui/react'
+
+const PopupExamplePosition = () => (
+  <Grid columns="repeat(3, 30px)" variables={{ padding: '30px', gridGap: '80px' }}>
+    <Popup
+      position="top-start"
+      trigger={<Button icon="expand" />}
+      content="Popup in top-start position"
+    />
+    <Popup position="top" trigger={<Button icon="expand" />} content="Popup in top position" />
+    <Popup
+      position="top-end"
+      trigger={<Button icon="expand" />}
+      content="Popup in top-end position"
+    />
+    <Popup
+      position="bottom-start"
+      trigger={<Button icon="expand" />}
+      content="Popup in bottom-start position"
+    />
+    <Popup
+      position="bottom"
+      trigger={<Button icon="expand" />}
+      content="Popup in bottom position"
+    />
+    <Popup
+      position="bottom-end"
+      trigger={<Button icon="expand" />}
+      content="Popup in bottom-end position"
+    />
+    <Popup
+      position="before-start"
+      trigger={<Button icon="expand" />}
+      content="Popup in before-start position"
+    />
+    <Popup
+      position="before"
+      trigger={<Button icon="expand" />}
+      content="Popup in before position"
+    />
+    <Popup
+      position="before-end"
+      trigger={<Button icon="expand" />}
+      content="Popup in before-end position"
```

Comment:
It would be great if the popup would be taller. It took me 10 minutes to see the different among `before-*` positions. (Two lines of text do the trick for me)

Reply:
good point, fixed, I'll update screenshots in description